spec/db/database.yml

```diff
@@ -21,3 +21,4 @@ mysql2:
   <<: *common
   adapter: mysql2
   username: root
+  host: 127.0.0.1
```

Comment:
The MySQL tests would not work without this because localhost tells the mysql client to use a local socket which won't work in github actions.